.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,80 @@
+name: 🐞 Bug report
+description: Create a report to help us improve
+title: "[BUG]"
+
+body:
+- type: markdown
+  attributes:
+    value: |
+      🧐 **Guidelines:**
+
+      - Search through [existing issues](https://github.com/CCBlueX/LiquidBounce/issues?q=is%3Aopen+is%3Aissue) first to ensure that this bug has not been reported before.
+      - Write a descriptive title for your issue. Avoid generic or vague titles such as "Something's not working" or "A couple of problems".
+      - Keep your issue focused on one single problem. If you have multiple bug reports, please create separate issues for each of them.
+      - Provide as much context as possible in the details section. Include screenshots, screen recordings, links, references, or anything else you may consider relevant.
+
+- type: input
+  attributes:
+    label: LiquidBounce Build Version
+    description: Which version of LiquidBounce does this bug affect?
+    placeholder: e.g. b61
+  validations:
+    required: true
+    
+- type: dropdown
+  attributes:
+    label: Operating System
+    description: On which OS is LiquidBounce running?
+    multiple: true
+    options:
+    - iOS
+    - Linux
+    - Windows 10
```

Comment:
You might as well add Windows 11 since it's coming out soon.

Reply:
Since it not released and also the Beta not even started yet, I think it would be better to add it later.

Reply:
Yes, it's not officially released RTM